src/Microsoft.Dnx.Tooling/Program.cs

```diff
@@ -80,7 +80,28 @@ public int Main(string[] args)
             WrapConsoleCommand.Register(app, reportsFactory);
             FeedsConsoleCommand.Register(app, reportsFactory);
 
-            return app.Execute(args);
+            try
+            {
+                return app.Execute(args);
+            }
+            catch (CommandParsingException ex)
+            {
+                AnsiConsole.GetError(_runtimeEnv.OperatingSystem == "Windows").WriteLine(ex.Message.Red().Bold());
+                ex.Command.ShowHelp();
+                return 1;
```

Comment:
I'm going to leave the -1 return codes for now just for consistency. The -1 should probably be all changed in one batch.